clang/test/dpct/launch-kernel-cooperative-usm.cu

```diff
@@ -24,6 +24,23 @@ __global__ void kernel(int *d, cudaTextureObject_t tex) {
   tex1D(d + gtid, tex, gtid);
 }
 
+// CHECK: void kernel_wrapper(int * d ,dpct::image_wrapper_base_p tex) {
```

Comment:
for new generate kernel_wrapper,  let's add a comment line to tell
// Auto generated SYCL kernel wrarpper used to ....